### DISCUSSION TITLE
how to bind to a attachedproperty in wpf

### DISCUSSION THREAD

**Discussion by wobushiafa** [reactiveui/ReactiveUI#3614] [General]
PasswordBoxHelper.Password

**Comment by glennawatson** [comment]
More detailed required.

**Comment by ChrisPulman** [comment]
I would create a ReactiveCommand and pass the Password control to the Parameter property of a BindCommand. The name you have provided suggests that its a custom control and not the default Wpf Password control.
If you provide us more information about exactly what you want to achieve then we can provide you the best practice methods.
